PUMI/globals.py

```diff
@@ -0,0 +1,6 @@
+from configparser import SafeConfigParser
+import PUMI
+import os
+
+cfg_parser = SafeConfigParser()
+cfg_parser.read(os.path.join(os.path.dirname(PUMI.__file__), 'settings.ini'))
```

Comment:
The config file should be searched for in the working directory, if not specified otherwise.
I now accept this, but we will have to sort this out later.